pkg/deploy/deployment_keycloak.go

```diff
@@ -365,19 +411,24 @@ func NewKeycloakDeployment(cr *orgv1.CheCluster, cheFlavor string, cheCertSecret
 			" && echo \"feature.token_exchange=enabled\nfeature.admin_fine_grained_authz=enabled\" > /opt/eap/standalone/configuration/profile.properties  " +
 			" && sed -i 's/WILDCARD/ANY/g' /opt/eap/bin/launch/keycloak-spi.sh && /opt/eap/bin/openshift-launch.sh -b 0.0.0.0"
 	}
+	args := []string{"-c", command}
+
+	if clusterDeployment != nil {
+		args = clusterDeployment.Spec.Template.Spec.Containers[0].Args
```

Comment:
The reason is the following:
`Args` field contain autogenerated passwords [1] if don't override this value will will continuously deploy keycloak.
 
[1] https://github.com/eclipse/che-operator/blob/ab/refactoring/pkg/deploy/deployment_keycloak.go#L38

Reply:
I'd rather get only the truststore password from the cluster keycloak deployment and use it in new args, instead of always overriding the args with the previous ones.
It seems that the current way will never update the commands if ever something has to be changed / fixed in the command line in future updates.